scrapli_community/nokia/sros/nokia_sros.py

```diff
@@ -73,6 +75,30 @@
     ),
 }
 
+CLASSIC_ARAM_PRIVILEGE_LEVELS = {
+    "exec": (
+        PrivilegeLevel(
+            pattern=r"^(.*)[a-zA-Z0-9_\:\-\>]*[\#\$] ?$",
```

Comment:
same question about the priv levels as in the other pr

Reply:
Same answer as the other PR. What do you suggest here? Taking in consideration that i would like to use send_config for all the drivers in my code.

Reply:
Removed config mode code. Pleease review.